filter_plugins/openshift_master.py

```diff
@@ -501,7 +501,36 @@ def validate_pcs_cluster(data, masters=None):
                 valid = False
         return valid
 
+    @staticmethod
+    def certificates_to_synchronize(hostvars):
```

Comment:
ah, not sure how I didn't catch that considering I verified the cert names used with the version conditions....